src/shared/form_fields/member.js

```diff
@@ -149,12 +142,13 @@ const validate = (values) => {
   const add_test = (tests, key) =>
     assoc(key, options[key] ? selected : exists, tests)
 
-  const req = id ? required :
-    new_required.concat(
+  const req = (id ? required : new_required)
+    .concat(
```

Comment:
@rjmk I did this as I thought if you were changing the membership_type to a life one on admin view you would want to put in a life_payment_date. I was also trying to work out how to make the date_membership_type_changed required only if the membership_type field had been touched and changed. I couldn't work it out though. Any ideas?